include/deal.II/lac/precondition.h

```diff
@@ -476,6 +476,29 @@ class PreconditionJacobi : public PreconditionRelaxation<MatrixType>
 {
 public:
   /**
+   * AdditionalData to store relaxation parameter.
+   */
+  struct AdditionalData
```

Comment:
I think it would be enough to do
```
struct AdditionalData : public PreconditionRelaxation<MatrixType>::AdditionalData
{};
```
Then initialize below becomes much simpler too (just pass ``parameters``).